File: app/src/main/java/com/intacta/sosviagens/Beans/Suggest.java
package com.intacta.sosviagens.Beans;

public class Suggest extends Road {

    public Suggest() {
    }


    public void setDia(String dia) {
        this.dia = dia;
    }

    private String dia;

    public Suggest(String rodovia, String concessionaria, String telefone, String id,String dia) {
        super(rodovia, concessionaria, telefone, id);
        this.dia = dia;
    }
}

File: app/src/main/java/com/intacta/sosviagens/model/Beans/Suggest.kt
package com.intacta.sosviagens.model.Beans

class Suggest : Road {

    private var dia: String? = null

    constructor() {}


    fun setDia(dia: String) {
        this.dia = dia
    }

    constructor(rodovia: String, concessionaria: String, telefone: String, id: String, dia: String) : super(rodovia, concessionaria, telefone, id) {
        this.dia = dia
    }
}

